pkg/k8s/run.go

```diff
@@ -67,10 +78,12 @@ func run(ctx context.Context, s *scanner, opt cmd.Option, artifacts []*artifacts
 		return xerrors.Errorf("k8s scan error: %w", err)
 	}
 
-	if err = write(report, pkgReport.Option{
```

Comment:
L53 need to handle error on defer `runner.Close()` wrap it with func and print log